src/audio/src/src_common.c

```diff
@@ -644,7 +644,7 @@ int src_reset(struct processing_module *mod)
 	return 0;
 }
 
-int src_free(struct processing_module *mod)
+__cold int src_free(struct processing_module *mod)
```

Comment:
commit desc says coefficients, but at end of this very long diff, a few functions move as well...? Seems ok, but slighly out of sync with git description.